<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToKeyConfig</class>
 <widget class="QWidget" name="ToKeyConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string extracomment="ToKey conf dialog, label for X symbol qlineedit">Convert to this Key *:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cfg_keysymbol">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cfg_eventtype">
     <property name="title">
      <string extracomment="ToKey conf dialog: label for groupButton">Event Type</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="cfg_eventtype_keypress">
        <property name="text">
         <string extracomment="ToKey conf dialog: label for checkbox">keypress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cfg_eventtype_keyrelease">
        <property name="text">
         <string extracomment="ToKey conf dialog: label for checkbox">keyrelease</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>* please remember to update settings on all tabs (keypress AND keyrelease)</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
